liberty-support-feature/src/feature/resources/arquillian-liberty-support.mf

```diff
@@ -1,8 +1,7 @@
 IBM-Feature-Version: 2
 IBM-ShortName: arquillian-support-jakarta-2.0
 Subsystem-Content: arquillian-liberty-support-jakarta; version=${parsedVersion.osgiVersion},
- com.ibm.wsspi.appserver.webBundle-1.0; type="osgi.subsystem.feature",
- com.ibm.websphere.appserver.servlet-5.0; ibm.tolerates:="5.0,6.0"; type="osgi.subsystem.feature"
+ com.ibm.websphere.appserver.localConnector-1.0; type="osgi.subsystem.feature"
```

Comment:
Should the version of this feature change since its implementation is? Or will that not matter to the end user?

Reply:
Yeah with this impl the code can work with Java EE 7, 8, 9 and 10.  I don't know we need a new version of it.  It will work for EE 9 and EE 10.